Makefile

```diff
@@ -0,0 +1,10 @@
+#/bin/bash
+
```

Comment:
As far as I can see, the build you linked is only when it is merged into master as you can see here: https://travis-ci.org/pierrre/imageserver/jobs/49433875#L75

If you look here, it also builds it from my repository: https://travis-ci.org/ruflin/gotestcover/jobs/76972333#L66

This makes it possible that I also use travis-ci myself with the repository and have the tests before I submit a pull request.